src/ggplotnim/ggplot_styles.nim

```diff
@@ -20,6 +20,10 @@ const LineDefaultStyle = Style(lineWidth: 1.0,
                                size: 5.0, # used to draw error bar 'T' horizontal
                                color: grey20,
                                fillColor: transparent)
+const RibbonDefaultStyle = Style(lineWidth: 1.0,
+                               lineType: ltNone,
+                               color: transparent,
+                               fillColor: grey20)
```

Comment:
I would propose default `fillColor` of 
`const grey70 = color(0.7, 0.7, 0.7)`, 
which you could add above these default styles definitions. The other consts are defined in `ginger` at the moment. This is pretty annoying, which is why I wanted to change it anyways. At some point it's probably wise to put all those consts for colors (and possibly way more?) into this repository in its own file or something.

With this default grey it would look nice even without alpha.